src/main/java/christmas/constants/EventType.java

```diff
@@ -0,0 +1,29 @@
+package christmas.constants;
+
+public enum EventType {
+    STAR("별", 5000),
+    TREE("트리", 10000),
+    SANTA("산타", 20000);
+
+    private final String name;
+    private final int nedeed;
+
+    EventType(final String name, final int nedeed) {
+        this.name = name;
+        this.nedeed = nedeed;
+    }
+
+    public static String findEventByMoney(final int discountAmonut) {
+        if (discountAmonut < STAR.nedeed) {
+            return "없음";
+        }
+        if (discountAmonut < TREE.nedeed) {
+            return STAR.name;
+        }
+        if (discountAmonut < SANTA.nedeed) {
+            return TREE.name;
+        }
+        return SANTA.name;
+    }
```

Comment:
```suggestion
    public static String findEventByMoney(final int discountAmonut) {
        for(EventType eventType : EventType.values()){
            if(discountAmount >= eventType.nedeed){
                return eventType.name;
            }
        }
        return "없음";
    }
```

Enum이 순서를 보장하기 때문에 type 선언시에 SANTA, TREE, STAR 순서로 선언해주시면 차례대로 반환 가능하기 때문에 위 코드 처럼 구현해볼 수 있을 것 같아요! 제가 스트림을 잘 못써서 반복문을 사용했지만 스트림을 이용하면 더 간단하게 줄일 수 있을 것 같아요!!😄